connector/src/main/scala/com/microsoft/kusto/spark/datasink/Writers.scala

```diff
@@ -62,7 +62,7 @@ case class EscapedWriter(out: java.io.Writer) extends Writer {
 
   def write(str: String): Unit ={
     for (c <- str) {
-      val escaped = EscapedWriter.escapeTable(c)
+      val escaped =  if (c > 127) 0 else EscapedWriter.escapeTable(c)
```

Comment:
So we are dumping all characters over 127? I thought we support UTF8

Reply:
every character that needs escaping is less than 127  ..